ci/infra/testrunner/skuba/skuba.py

```diff
@@ -26,28 +28,16 @@ def _verify_skuba_bin_dependency(self):
             raise FileNotFoundError("skuba not found at {}".format(self.binpath))
 
     def _verify_bootstrap_dependency(self):
-        if not os.path.exists(os.path.join(self.conf.workspace, "test-cluster")):
-            raise ValueError("test-cluster not found. Please run bootstrap and try again")
+        if not os.path.exists(self.cluster_dir):
+            raise ValueError(f'cluster {self.cluster} not found at {self.workdir}.'
+                             ' Please run bootstrap and try again')
 
     @staticmethod
     @step
     def cleanup(conf):
         """Cleanup skuba working environment"""
-
-        # TODO: check why (and if) the following command is needed
-        # This is pretty aggressive but modules are also present
-        # in workspace and they lack the 'w' bit so just set
-        # everything so we can do whatever we want during cleanup
-        Utils.chmod_recursive(conf.workspace, 0o777)
```

Comment:
Love to see that gone! :+1: Removing this line has been in my git stash pile since I started in working with testrunner :smile: